config/serverless.es.yml

```diff
@@ -17,17 +17,17 @@ xpack.osquery.enabled: false
 ## Fine-tune the search solution feature privileges. Also, refer to `serverless.yml` for the project-agnostic overrides.
 xpack.features.overrides:
   ### Dashboards feature is moved from Analytics category to the Search one.
-  dashboard.category: "enterpriseSearch"
+  dashboard.category: 'enterpriseSearch'
```

Comment:
do we need these quote changes?

Reply:
We do not need these quote changes. These go added when I rand `lint` command so I assumed something changed with the eslint configuration which forced the file to update.